io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/reader/ReaderSettings.java

```diff
@@ -190,9 +199,14 @@ else if (skipType) {
 		else {
 			skipNlines = 0;
 		}
+
+		// read dateFormat
+		String dateFormatString = reader.getParameter(PARAMETER_DATE_FORMAT).as(String.class);
```

Comment:
@stempler Do you have any idea why this is consistently null? Am I overlooking something in the setup? Thank you.

Reply:
In the test it's working, as I set it before. 
But when using hale studio it's null.

Reply:
Since there is no UI to configure it, this is to be expected.
If you save a project with your loaded data you could manipulate the project file to include a value for this setting, if you want to test with hale studio.